ajax/libs/sscaffold-css/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "sscaffold-css",
+  "version": "0.1.0",
```

Comment:
The auto-update bot will provide this attr, please don't include it unless you wish to include all the needed files for this version in the PR.